bin/ingest_single_footprint.sh

```diff
@@ -48,7 +47,6 @@ cat << EOF
     min_zoom=greatest(0, approximate_zoom - 4),
     max_zoom=20,
```

Comment:
Ah. I think ETOPO is the only one with that problem. It shows itself pretty quickly.

`approximate_zoom + 8` is probably the cut-off (that's when a pixel gets to the point of taking up a whole tile).